internal/manifest/report_test.go

```diff
@@ -267,6 +281,154 @@ var reportTests = []struct {
 	add:     []sliceAndEntry{{entry: sampleDir, slice: oneSlice}},
 	mutate:  []*fsutil.Entry{&sampleDir},
 	err:     `cannot mutate path in report: /example-dir/ is a directory`,
+}, {
+	summary: "Hard link to regular file",
+	add: []sliceAndEntry{
+		{entry: sampleFile, slice: oneSlice},
+		{entry: sampleHardLink, slice: oneSlice}},
+	expected: map[string]manifest.ReportEntry{
+		"/example-file": {
+			Path:       "/example-file",
+			Mode:       0777,
+			SHA256:     "example-file_hash",
+			Size:       5678,
+			Slices:     map[*setup.Slice]bool{oneSlice: true},
+			HardLinkID: 1,
+		},
+		"/example-hard-link": {
+			Path:       "/example-hard-link",
+			Mode:       0777,
+			SHA256:     "example-file_hash",
+			Size:       5678,
+			Slices:     map[*setup.Slice]bool{oneSlice: true},
+			HardLinkID: 1,
+		},
+	},
+}, {
+	summary: "Hard link to symlink",
```

Comment:
Let's actually remove this use-case because the logic is the same at the report level, right? This is only meaningful at the `create` level and at the `slicer`/`extractor` level for integration, here it is just another field.

Reply:
Yes. Removed.